moqt-server/src/modules/handlers/subscribe_error_handler.rs

```diff
@@ -0,0 +1,376 @@
+use anyhow::{bail, Result};
+use moqt_core::pubsub_relation_manager_repository::PubSubRelationManagerRepository;
+use moqt_core::{
+    constants::StreamDirection,
+    messages::{control_messages::subscribe_error::SubscribeError, moqt_payload::MOQTPayload},
+    MOQTClient, SendStreamDispatcherRepository,
+};
+
+pub(crate) async fn subscribe_error_handler(
+    subscribe_error_message: SubscribeError,
+    pubsub_relation_manager_repository: &mut dyn PubSubRelationManagerRepository,
+    send_stream_dispatcher_repository: &mut dyn SendStreamDispatcherRepository,
+    client: &mut MOQTClient,
+) -> Result<()> {
+    tracing::trace!("subscribe_error_handler start.");
+
+    tracing::debug!("subscribe_error_message: {:#?}", subscribe_error_message);
+
+    let upstream_session_id = client.id;
+    let upstream_subscribe_id = subscribe_error_message.subscribe_id();
```

Comment:
I don't know the detailed reason, because I was not the one who first adopted this format.
I think it is probably to handle the variable as read-only.

Reply:
OK, Let's decide that the properties of the struct should be accessed from only getter function.